illumos-utils/src/running_zone.rs

```diff
@@ -564,6 +564,9 @@ pub enum InstallZoneError {
         #[source]
         err: crate::zone::AdmError,
     },
+
+    #[error("Failed to find zone image '{image}' from `{paths:?}'")]
```

Comment:
Nit - mismatched quotes; maybe
```suggestion
    #[error("Failed to find zone image '{image}' from '{paths:?}'")]
```

or even

```suggestion
    #[error("Failed to find zone image '{image}' from {paths:?}")]
```

since the debug impl will enclose it in brackets?

Reply:
Done